### DISCUSSION TITLE
How to sign up for student developer pack

### DISCUSSION THREAD

**Discussion by sidehustlesoftwares** [community/community#64823] [GitHub Education] [answered]
### Hi everyone,

Hi everyone,

How can i sign up for the GitHub student developer pack if i don't have a school issued ID.

**Comment by arafajs** [comment, marked as answer]
You can do this by providing other documentation that proves your current student status. This documentation can include:

- A current transcript
- A letter on school letterhead
- A screenshot of your school's website that shows your name and enrollment status
- Any other documentation that you think is relevant

Then:

1. Go to GitHub Education and click on "Get benefits" in the top right corner.
2. Under "Which best describes your academic status?", select "Student".
3. If you have a school-issued email, select (or add it). Please note that email addresses must be verified and any addresses with plus signs (“+”) are not eligible.
4. If you don't have a school-issued email, follow the prompts to fill out additional information. You will be asked to upload documentation that proves your current student status.
5. Click on "Submit".

**Comment by Daniyal-Aslam** [comment]
Hello,

I'm encountering difficulties verifying my school-issued email, [daniyalaslam@students.usindh.edu.pk](mailto:daniyalaslam@students.usindh.edu.pk), for the GitHub Student Developer Pack. I'm a student at the University of Sindh Jamshoro.

I'm still getting the same error it says (We require applicants to use a school-issued email address to apply.) even after verifying it from my github.

Can anyone please assist in resolving this issue?

**Comment by TaimoorCodes** [comment]
[Here](https://medium.com/@taimooor/how-to-get-your-github-student-developer-pack-5cab9f600da9), this might help.
